experimental/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to experimental packages in this project will be documented
 ### :boom: Breaking Change
 
 * chore(instrumentation-grpc): Drop support for package `grpc`. [#3807](https://github.com/open-telemetry/opentelemetry-js/pull/3807) @llc1123
+* chore(instrumentation-grpc): cleanup remnants of grpc-native support. [#3886](https://github.com/open-telemetry/opentelemetry-js/pull/3886) @llc1123
```

Comment:
This doesn't really describe what is breaking about the change. Users look at the changelog to see what breaking changes may affect them

Reply:
Details added.